core/templates/results.html

```diff
@@ -70,33 +80,100 @@
 
 {% block extra_js %}
 <script>
-    var labels = $.parseHTML('{{ names }}');
-    labels = labels[0].textContent;
-    labels = labels.split(';');
-    var known = {{ known }};
-    var imgs = '{{ mugs }}';
-    imgs = imgs.split(';');
-
-	var barChartData = {
-		labels : labels,
-		datasets : [
-			{
-				fillColor : "rgba(51, 102, 153,0.5)",
-				strokeColor : "rgba(220,220,220,0.8)",
-				highlightFill: "rgba(220,220,220,0.75)",
-				highlightStroke: "rgba(220,220,220,1)",
-				data : known
-			}
-		]
-
-	};
-	window.onload = function(){
-		var ctx = document.getElementById("graph").getContext("2d");
-		window.myBar = new Chart(ctx).Bar(barChartData, {
+    var graphId = ['#whoYouKnow', '#globalTen'];
+    var graphIdx = 0;
+
+    function allActive(){
+        for(var x = 0; x < graphId.length; x++){
+            $(graphId[x]).addClass('active');
+        }
+    }
+
+    function showGraph(){
+        $(graphId[graphIdx]).removeClass('active');
+        if(graphIdx != (graphId.length -1))
+            graphIdx++;
+        else
+            graphIdx = 0;
+    }
+
+    $(document).ready(function () {
+        $('.carousel').carousel({
+            interval: false
+        });
+
+        $(".carousel").on('slid.bs.carousel', function (evt) {
+            $('#graph').css('height','400px');
+            $('#globalTenGraph').css('height','400px');
+
+            allActive();
+            loadCharts();
+            showGraph()
```

Comment:
Pretty much how this is works is on the end of the carousel slide event:
1) it makes sure all the carousel "pages" are 400px in height
2) then forces them to be all active on the screen
3) Reloads the graphs + animations
4) Then shows correct graph by index